tests/ByCodeAnalysis/ArtisanFieldsTest.php

```diff
@@ -48,13 +48,15 @@ public function testArtisanTsModel(): void
 
         $fieldsInJson = Fields::public()->asArray();
 
-        $matches->forEach(function (Detail $detail) use (&$fieldsInJson): void {
+        foreach ($matches as $detail) {
+            static::assertInstanceOf(Detail::class, $detail); // Silence PHPStan
```

Comment:
Outside tests I'd probably go with a `@var` annotation...

Reply:
Yes, my mistake. That's very good idea.